source/dom/dispatcher.js

```diff
@@ -54,8 +54,20 @@
 		/**
 		* @private
 		*/
+		platformSpecific: function() {
+			var d = enyo.dispatcher,
+				webosWindowEvents = ["focus", "blur"];
+			if (enyo.platform.webos >= 4) {
+				d.windowEvents.concat(webosWindowEvents);
```

Comment:
Array.prototype.concat method returns the new array and does not alter the arrays you're concatenating, so, this line is doing work but you aren't saving the returned array?